.github/workflows/cypress.yml

```diff
@@ -0,0 +1,64 @@
+name: Cypress
+
+on: deployment_status
+
+jobs:
+  cypress:
+    if: "!contains(github.event.head_commit.message, 'chore(release): publish') && github.event.deployment_status.state == 'success'"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2-beta
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Get yarn cache
+        id: yarn-cache
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - uses: actions/cache@v1
+        with:
+          path: ${{ steps.yarn-cache.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+      - name: restore lerna
+        uses: actions/cache@master
+        with:
+          path: |
+            node_modules
+            */*/node_modules
+          key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
+      - name: Install dependencies
+        run: yarn install
+      - name: Print URL
+        run: echo Testing URL ${{ github.event.deployment_status.target_url }}
+      - name: Run cypress
+        run: yarn ci:website
+        env:
+          APPLITOOLS_API_KEY: ${{ secrets.APPLITOOLS_API_KEY }}
+          CYPRESS_BASE_URL: ${{ github.event.deployment_status.target_url }}
```

Comment:
Are these still running on the deployment or the website running in the ci box?

Reply:
They're running on the Vercel deployment. Changing that to run just on the website running on the CI box might speed up the tests since we wouldn't have to wait for the deployment.

Reply:
Yeah, that might have been a miscommunication. When asking for the way to run tests locally, I was expecting to just use the same script for ci too

Reply:
In my mind it made sense to test the production version of the site, which led to testing the preview deployment. That said I could alter the test:website script to be something like:

```
start-server-and-test 'yarn build:website' 'yarn serve:website' http://localhost:9000 'yarn run cypress run --spec './cypress/integration/**/*.spec.ts''
```

That will also simplify reporting back to github, as I could run the action on `push` instead of `deployment_status`.

Reply:
Are you only reporting back to the checks api for the applitools report?

Reply:
I had to report it to the checks api, because the run wasn't really being tied to the PR. See here: https://share.getcloudapp.com/4guJ2ZBY

Since Vercel handled the deployment status, it kicked off the Cypress action and somehow lost the tie to the PR. I'm testing my above idea about changing the script locally, and that seems to be working better. I don't think I'd need to report back to the checks api since the action runs on `pull_request`.

Reply:
I see, so yeah, that makes sense then to me not to use the deployment, just for not having to do a custom report back to the checks API.